### PR TITLE
Add badge image to the book details (Book details part 4) 

### PR DESCRIPTION
**Pull request by JuanGutierrez43**
## Summary 

This will add the badge image and set it in the top right corner of the book image. For this, I used transform to rotate the image like the card and then position to set it all over the book image. 

**New features**

- Add image badge.png
- Set the image over the book image using transform and position.

## Screenshots 
Screenshot from 2020-03-05 11-27-54

## Cards

https://trello.com/c/keek9FQ9/29-detalle-de-libro-iv-badge

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,6 +13,7 @@
 <body>
   <div class="book-cover-page">
     <img class="book-img" src="images/book-cover.png" alt="Book Cover">
+    <img class="book-badge" src="images/badge.png" alt="Book Badge">
```
Comment (eledwinn) [inline index.html]: Please, use pseudo-elements to add decorative elements.

File: scss/detail.scss [hunk 2]
```diff
@@ -4,6 +4,15 @@
   font-family: Roboto;
 }
 
+.book-badge {
+    height: 120px;
+    width: 100px;
+    transform: rotate(1deg);
```
Comment (eledwinn) [inline scss/detail.scss]: Cool!!!

**Review by eledwinn — CHANGES_REQUESTED** [review]

File: scss/detail.scss [hunk 3]
```diff
@@ -4,6 +4,15 @@
   font-family: Roboto;
 }
 
+.book-badge {
```
Comment (CrisVega08) [inline scss/detail.scss]: When you use pseudo element this class disappear

**Review by CrisVega08 — CHANGES_REQUESTED** [review]

File: scss/detail.scss [hunk 4]
```diff
@@ -26,6 +17,17 @@
   box-shadow: 4px 4px 10px 0 $black-shadow;
 }
 
+.book-cover-page::after {
+  background: url("images/badge.png") no-repeat;
+  content: "";
```
Comment (eledwinn) [inline scss/detail.scss]: Please use single quotes 😄

File: scss/detail.scss [hunk 5]
```diff
@@ -26,6 +17,17 @@
...
+  height: 120px;
+  width: 100px;
```
Comment (eledwinn) [inline scss/detail.scss]: and sort props :P

**Review by eledwinn — CHANGES_REQUESTED** [review]

**Review by eledwinn — APPROVED** [review]

File: scss/detail.scss [hunk 6]
```diff
@@ -9,12 +9,23 @@
...
   display: flex;
-  margin: 50px 0 0 50px;
   height: 416px;
+  margin: 50px 0 0 50px;
   width: 900px;
-  background-color: $white;
-  box-shadow: 4px 4px 10px 0 $black-shadow;
+}
+
+.book-cover-page::after {
```
Comment (CrisVega08) [inline scss/detail.scss]: nice but you can made pseudo element inside of book-cover-page something like:
`
.book-cover-page {
  background-color: $white;
  box-shadow: 4px 4px 10px 0 $black-shadow;
  display: flex;
  height: 416px;
  margin: 50px 0 0 50px;
  width: 900px;

  &::after {
    background: url('images/badge.png') no-repeat;
    content: '';
    height: 120px;
    left: 250px;
    position: absolute;
    top: 50px;
    transform: rotate(1deg);
    width: 100px;
  }
}
`

**Review by CrisVega08 — APPROVED** [review]

**Pull request merged**